informus/content/all_floorplan_titles_and_legends.md

```diff
@@ -113,6 +113,15 @@ Add legend item - dark grey bed, label as 'incomplete data'
 
 ![image](https://github.com/user-attachments/assets/c00a4de3-8a86-486c-825d-0ea9542737d2)
 
+Legend Labeling wording change: 
+
+CAM-ICU positive (no RASS) to CAM-ICU positive (RASS overdue)
+
+and 
+
+CAM-ICU negative (no RASS) to CAM-ICU negative (RASS overdue)
+
+
```

Comment:
Instead of creating a new key for the changes to be made, you can just make the changes directly in the relevant sections